03-csv-vector-raster-plotting.Rmd

```diff
@@ -325,7 +329,7 @@ plot(extent(aoiBoundary_HARV),
      lwd=6,
      col="springgreen")
 
-legend("left",
+legend("bottom",
```

Comment:
I did review all changes in local builds.  The solution to this problem is finding a way to fix it to not overlap as the preferred non-optional solution seems to be personal preference.  
When on the left the legend line symbol overlapped the boundary in a perpendicular angle while the words were in the interior white space.  I spent a while puzzling over what that element in the map was (not connecting the words to the overlapping green and purple "crosses"), prior to realizing it was the legend.  While bottom is no where near ideal, I found that since the words overlapped as well as the symbols that it was clear this wasn't a unique element but part of the legend.   

Issue calling for need to fix this:  https://github.com/data-lessons/NEON-R-Spatial-Vector/issues/78